scripts/zones/Riverne-Site_A01/npcs/Spatial_Displacement.lua

```diff
@@ -17,67 +17,11 @@ function onTrigger(player,npc)
         player:startEvent(3);
     elseif (id == base+2) then
         player:startEvent(4);
-    elseif (id == base+7) then
-        player:startEvent(7);
-    elseif (id == base+8) then
-        player:startEvent(8);
-    elseif (id == base+9) then
-        player:startEvent(9);
-    elseif (id == base+10) then
-        player:startEvent(10);
-    elseif (id == base+11) then
-        player:startEvent(11);
-    elseif (id == base+12) then
-        player:startEvent(12);
-    elseif (id == base+13) then
-        player:startEvent(13);
-    elseif (id == base+14) then
-        player:startEvent(14);
-    elseif (id == base+15) then
-        player:startEvent(15);
-    elseif (id == base+16) then
-        player:startEvent(16);
-    elseif (id == base+18) then
-        player:startEvent(18);
-    elseif (id == base+20) then
-        player:startEvent(20);
-    elseif (id == base+21) then
-        player:startEvent(21);
-    elseif (id == base+22) then
-        player:startEvent(22);
-    elseif (id == base+23) then
-        player:startEvent(23);
-    elseif (id == base+24) then
-        player:startEvent(24);
-    elseif (id == base+25) then
-        player:startEvent(25);
-    elseif (id == base+26) then
-        player:startEvent(26);
-    elseif (id == base+27) then
-        player:startEvent(27);
-    elseif (id == base+28) then
-        player:startEvent(28);
-    elseif (id == base+29) then
-        player:startEvent(29);
-    elseif (id == base+30) then
-        player:startEvent(30);
-    elseif (id == base+31) then
-        player:startEvent(31);
-    elseif (id == base+33) then
-        player:startEvent(21);
-    elseif (id == base+34) then
-        player:startEvent(22);
-    elseif (id == base+35) then
-        player:startEvent(23);
-    elseif (id == base+36) then
-        player:startEvent(24);
-    elseif (id == base+37) then
-        player:startEvent(25);
-    elseif (id == base+38) then
-        player:startEvent(26);
-    elseif (id == base+39) then
-        player:startEvent(27);
-    end
+    elseif (id - base <= 39) then
+        player:startEvent(id - base);
+    end;
+    
+end;
 
```

Comment:
Except for this random end; sitting here... this script works. All of the displacements teleported me to the proper location. (Haven't tested B01 yet)

Reply:
Looks like the end belongs there, but it got a semicolon added so git sees it as a new one.